crates/tokio-util/src/event_listeners.rs

```diff
@@ -1,46 +1,41 @@
-use tokio::sync::mpsc;
-use tokio_stream::wrappers::UnboundedReceiverStream;
+use tokio::sync::broadcast::{self, Sender};
+use tokio_stream::wrappers::BroadcastStream;
+use tracing::error;
 
-/// A collection of event listeners for a task.
-#[derive(Clone, Debug)]
+const DEFAULT_SIZE_BROADCAST_CHANNEL: usize = 2000;
+
+/// A bounded broadcast channel for a task.
+#[derive(Debug, Clone)]
 pub struct EventListeners<T> {
```

Comment:
I don't think the name still fits, this no longer wraps a list of channels.

I guess the naming was off originally, maybe this should just be `EventSender`?

Reply:
Sure, `EventSender` describes better the current implementation, done